package.xml

```diff
@@ -22,19 +22,25 @@ limitations under the License.
 	<summary>Apache SkyWalking PHP Agent.</summary>
 	<description>The PHP Agent for Apache SkyWalking, which provides the native tracing abilities for PHP project.</description>
 	<lead>
+        <name>Apache SkyWalking</name>
+		<user>Apache SkyWalking</user>
+        <email>dev@skywalking.apache.org</email>
+		<active>yes</active>
+	</lead>
+	<developer>
 		<name>jmjoy</name>
 		<user>jmjoy</user>
 		<email>jmjoy@apache.org</email>
 		<active>yes</active>
-	</lead>
-	<lead>
+	</developer>
+	<developer>
 		<name>Yanlong He</name>
 		<user>yanlong</user>
 		<email>yanlong@php.net</email>
 		<active>yes</active>
-	</lead>
-	<date>2022-09-04</date>
-	<!-- TODO -->
+	</developer>
+	<!-- TODO The date should be changed before release -->
```

Comment:
Let's add a kind of anchor here? And write a pre-release script to update this automatically?
Then put this script in the pre-release steps of the release guide(Like Rust's?)

Reply:
I think we can create a `package.tpl.xml` file and put `package.xml` to `.gitignore`, and use a script to generate the `package.xml`.

Because `<notes>` needs the changelog, so we can add a CI, trigger by the pre-release event, and auto fetch the changelog content to replace it.

@heyanlong What's your idea?

Reply:
I think tpl.xml should be good too. We use `makefile` as the default compiling entrance anyway.

Reply:
`Makefile` is occupied by PECL and phpize (auto generate), so we can't use it...

We can add a [[bin]] in Cargo.toml, and then call `cargo run` to generate the `package.xml`.

Reply:
Once we are clear about how to build in the docs, we should be good. Either makefile or cargo run is a good tooling.

Reply:
:ok:

Reply:
I also will generate the git tree file list for `<contents>`, now the field to be auto generated is `<date>`, `<version>`, `<notes>` and `<contents>`, alright?

Reply:
Things you plan to change before release, we should be good. I just don't want you to have to re-run all release processes, because of forgetting these steps.

Reply:
tpl.xml is a good idea

Reply:
:ok_hand: